lib/cdk-infra-stack.ts

```diff
@@ -601,18 +637,17 @@ export default class FhirWorksStack extends Stack {
             }),
             tracing: Tracing.ACTIVE,
         });
-        // pending port of hapi validator stack
```

Comment:
Remove it if it's not needed?

Reply:
If this is referring to the comment, it's already been removed.

Reply:
What about the commented out section below? Is it needed?